docs/23_gitignore.md

```diff
@@ -0,0 +1,44 @@
+## Configure Git to ignore files
+
+While working within your repository, you might notice files that your preferred editor or operating system have place in the repository. These files are typically things that you don't want git to track or include in the history of your project. Although you can manually ignore those files by never committing them to the history of the project, there is an easier way for you to ignore those files. Git enables you to ignore specific files, files types, and folders through the use of a text file you can add to your  repository titled `.gitignore`. This file instructs Git to stop tracking changes of certain files, which means that these files will not be pushed to GitHub from your local machine.
+
+**Files that you might want Git to ignore:**
+1. Secrets, e.g. `*.pem`, `*.key`
+1. Binary files, e.g. `*.pdf*`, `*.docx`
+1. Log files, e.g. `*.log`
+1. Build files, e.g. `Build/*`
```

Comment:
Maybe this section should be moved down below the steps to create an empty `.gitignore` file.